RELEASE_GUIDE.md

```diff
@@ -1,26 +1,49 @@
 # Release Guide (for maintainers)
 
+## Prerequisites
+
+- You will need a [Hackage](https://hackage.haskell.org/) account that has been invited to be a maintainer of the `purescript` package on Hackage. If you don't have one, create one and ask to be invited as a maintainer.
+- You will need an [NPM](https://www.npmjs.com/) account that has been invited to be a maintainer of the `purescript` package on NPM. If you don't have one, create one and ask to be invited as a maintainer.
+
 ## Before making a release
 
 - Check that there are no unintended breaking changes by compiling [the latest package set](https://github.com/purescript/package-sets/releases/latest)
-- Check that INSTALL.md is up-to-date
-- Regenerate LICENSE: `make license-generator` (see `license-generator/` for
-  details)
-- Write release notes
 
-Additionally, if there are any breaking changes, there are number of downstream
-projects who we should probably at least notify:
+```bash
+stack build
+mkdir wPackageSet
+pushd wPackageSet
+spago init
+spago upgrade-set
+# install all packages in the set
+spago install $(spago ls packages | cut -f 1 -d ' ' | tr '\n' ' ')
+
+stack exec purs compile $(spago sources | tr '\n' ' ')
+stack exec purs docs $(spago sources)
```

Comment:
Note: these two `stack exec` commands don't work. I'm not sure how to pass the source globs correctly to the resulting `purs` binary.

Reply:
If you run `stack exec -- bash` then you'll be dropped into a shell, so maybe that'd be easier?

Reply:
Yeah, I think `stack exec bash` followed by `spago build` and `spago docs` is probably easiest.

Reply:
I tried `stack exec bash` and it didn't drop into a shell. Found [this SO answer](https://stackoverflow.com/a/48014385) and think we should use this:
```bash
stack exec bash <<EOF
spago build
spago docs
EOF
```